Automattic/wp-calypso
# Deploy Woo branding update on some environments

## Proposed Changes

Enables Woo Branding update to staging, horizon, and wpcalypso environments

## Why are these changes being made?

To prepare for internal testing: pdibGW-3QO-p2#comment-2951

## Testing Instructions

1. Reviewing the code changes should be sufficient.
2. If you'd like to test the branding update, you can follow the instructions from any of these PRs:
    - https://github.com/Automattic/wp-calypso/pull/97318
    - https://github.com/Automattic/wp-calypso/pull/97152
    - https://github.com/Automattic/wp-calypso/pull/97058
    - https://github.com/Automattic/wp-calypso/pull/96963

## Pre-merge Checklist
